server/conf/i18n/messages.am

```diff
@@ -369,7 +369,6 @@ validation.phoneMustBeLocalToCountry=ያቀረቡት ስልክ የአገሩ አ
 # DATE QUESTION - text shown when answering a question where a user must select date. #
 #----------------------------------------------------------------------------------------------------------#
 
-validation.dateMisformatted=እባክዎ በዓዓዓዓ/ወወ/ቀቀ ቅርጸት ቀን ያስገቡ።
```

Comment:
question (blocking): do you need to update the translated files or should that happen automatically via transifex? I know additions are automatic, but i'm not sure what happens with deletions.

Reply:
I believe that they have to be deleted manually (see https://github.com/civiform/civiform/pull/7059 for another example)